rancher2/schema_node_template_azure.go

```diff
@@ -23,6 +23,7 @@ type azureConfig struct {
 	Location           string   `json:"location,omitempty" yaml:"location,omitempty"`
 	ManagedDisks       bool     `json:"managedDisks,omitempty" yaml:"managedDisks,omitempty"`
 	NoPublicIP         bool     `json:"noPublicIp,omitempty" yaml:"noPublicIp,omitempty"`
+	Nsg                string   `json:"nsg,omitempty" yaml:"nsg,omitempty"`
```

Comment:
Could you please update field name to capital letters to match rancher machine definition, https://github.com/rancher/machine/blob/master/drivers/azure/azure.go#L92 ??

```
-	Nsg                string   `json:"nsg,omitempty" yaml:"nsg,omitempty"`
+	NSG                string   `json:"nsg,omitempty" yaml:"nsg,omitempty"`
```